cmd/copyEnumeratorInit.go

```diff
@@ -57,7 +57,7 @@ func (cca *cookedCopyCmdArgs) initEnumerator(jobPartOrder common.CopyJobPartOrde
 	jobPartOrder.DestLengthValidation = cca.CheckLength
 	jobPartOrder.S2SInvalidMetadataHandleOption = cca.s2sInvalidMetadataHandleOption
 
-	traverser, err = initResourceTraverser(cca.source, cca.fromTo.From(), &ctx, &srcCredInfo, &cca.followSymlinks, cca.listOfFilesChannel, cca.recursive, getRemoteProperties, func(common.EntityType) {})
+	traverser, err = initResourceTraverser(cca.source, cca.fromTo.From(), &ctx, &srcCredInfo, &cca.followSymlinks, cca.listOfFilesChannel, cca.recursive, getRemoteProperties, false, func(common.EntityType) {})
```

Comment:
Minor: why not use cca.includeDirectoryStubs here, insteadof hard coding the "false"? (In case we move remove over to use initCopyEnumerator).

Reply:
Actually isn't this for blob? So logically it would use initCopyEnumerator already, since azbfs is handled in its own case.

---
In reply to: [433610287](https://github.com/Azure/azure-storage-azcopy/pull/1008#discussion_r433610287) [](ancestors = 433610287)

Reply:
(I stand corrected, forgot we had remove-specific enumerator init)

---
In reply to: [438900923](https://github.com/Azure/azure-storage-azcopy/pull/1008#discussion_r438900923) [](ancestors = 438900923,433610287)